app/src/main/java/com/arielwang/workoutlogger/database/daos/WorkoutDao.kt

```diff
@@ -0,0 +1,15 @@
+package com.arielwang.workoutlogger.database.daos
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.Query
+import com.arielwang.workoutlogger.database.model.WorkoutAddingFlow
+
+@Dao
+interface WorkoutDao {
+  @Query("SELECT * FROM WorkoutAddingFlow")
+  suspend fun getAllExercise(): List<WorkoutAddingFlow>
```

Comment:
This could be renamed to getAllWorkoutAddingFlow

Reply:
Thank you.